integration/oasis/solutionformat/doc.go

```diff
@@ -0,0 +1,7 @@
+/*
+package solutionformat defines internal foramt for recording single solution result.
+It isolates internal calculation logic and final restful response:
+- Internal algorithm will calculate result in this format.
+- In the end, oasis will use this format to generate restful response.
+*/
+package solutionformat
```

Comment:
I feel `solution` will be better, even it causes `solution.Solution`. How's your idea?

Reply:
I like shorter name.  

My original thinking is, `solution` is better contains not only `type definition` but also `solution logic`, which is how those type is constructed, such as the calculation logic.

Currently, I put constructing logic inside `stationgraph`, in the end of its calculation it will generate result in `solutionformat`, which could be dumped to external format.  That's what I try to limit scope with `solutionformat` or `solutiontype`

But as a reviewer, I think `solution` is more clear than `solutionservice` or `solutiontype`.

Reply:
This issue I think is worth to record: Avoid to use package name as variable name, otherwise you will get strange errors.

After I rename package name from `solutionformat` to `solution`, there are some code use this package create variable `solution` there.  (this code is not uploaded, they exists in the big pull request https://github.com/Telenav/osrm-backend/pull/203)
```
solution := &solution.Solution{}
solution.ChargeStations = make([]*solution.ChargeStation, 0)
```
I met following error:
![image](https://user-images.githubusercontent.com/16873751/76246249-8fc51e80-61fa-11ea-94a4-3e99c4d42ffe.png)

I think compiler confuse on whether `solution` is a package name or a variable name.  After I rename variable `solution` to `sol` problem solved.  But it took me a while to figure it out.  I original thought it is related with package renaming.